pkg/webui/oauth/views/app/index.js

```diff
@@ -58,8 +60,8 @@ export default class OAuthApp extends React.PureComponent {
         <Provider store={store}>
           <Init>
             <Helmet
-              titleTemplate="%s - Account - TTN Stack"
-              defaultTitle="TTN Stack Account"
+              titleTemplate={`%s - Account - ${env.site_name}`}
```

Comment:
I think also here, we should not hard code anything. If the `site_title` is not set, let's just omit it altogether. 
```suggestion
              titleTemplate={`%s - ${env.site_title ? `${env.site_title} -` : ''}${env.site_name}`}
```